.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint and Test
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test
+    timeout-minutes: 2
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          ref: ${{ github.ref }}
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
```

Comment:
Thoughts on doing a matrix to test all our supported versions?

Reply:
How far back do we support? 10?

Reply:
I'm not sure, but that's what the travis config suggests.

Reply:
Do you happen to know any examples for how to run the same job on each version? I'm guessing I give a matrix of inputs to it, but I don't know exactly how to write that

Reply:
Yep: https://github.com/neverendingqs/serverless-dotenv-plugin/blob/8ef262f0b0369640ae8cf61e5aed3a45c290b853/.github/workflows/ci.yaml#L10-L19

I think it's in the docs as well, but I forgot where (maybe the workflow syntax one).